src/select.c

```diff
@@ -6313,6 +6313,7 @@ static int selectExpander(Walker *pWalker, Select *p){
   return WRC_Continue;
 }
 
+#ifndef _FREEBSD_KERNEL
```

Comment:
I would change the SQLITE_DEBUG in whatever the root include file (.h) is and put it and other directives you're #ifdef'ing out under a single _FREEBSD_KERNEL block.

Reply:
Just removed the guards -- the SQLITE_DEBUG was not defined, and because of the -Wno-undef, it defaults to 0, which is fine.